projects/igniteui-angular/src/lib/tabs/tabs.directive.ts

```diff
@@ -120,8 +122,11 @@ export abstract class IgxTabsDirective extends IgxCarouselComponentBase implemen
     private _itemChanges$: Subscription;
 
     /** @hidden */
-    constructor(builder: AnimationBuilder, cdr: ChangeDetectorRef, public dir: IgxDirectionality) {
-        super(builder, cdr);
+    constructor(
+        @Inject(IgxAngularAnimationService) animationService: AnimationService,
```

Comment:
```suggestion
        @Inject(IgxAngularAnimationService) private animationService: AnimationService,
```

Reply:
this is already marked as private in base class